pro/pro-setup.rst

```diff
@@ -67,26 +67,37 @@ Upgrade to Pro from Community Edition
 
 You can upgrade from an existing community server to a pro server with this script. Follow the prompts to setup a pro server and the script will make the necessary changes to your netmaker.env file and grab the pro docker-compose.override.yml file.
 
-If you prefer to upgrade manually, go to your netmaker server and add the following to your netmaker.env file.
+If you prefer to upgrade manually, go to your netmaker server and add the following to your netmaker.env file:
 
-.. code-block:: yaml
+.. code-block::
+
+    LICENSE_KEY=“<license key>”
+    NETMAKER_TENANT_ID="<tenant id>"
+
+Make sure that it should look something like this:
+
+.. code-block::
 
-    LICENSE_KEY: “<license key>”
-    NETMAKER_TENANT_ID: "<tenant id>"
+    LICENSE_KEY=mylicensekeymylicensekeymylicensekeymylicensekeymylicensekey
+    NETMAKER_TENANT_ID=myguidte-nant-idmy-guid-tenantidmygu
```

Comment:
why random text, is this required ?

Reply:
no. I just thought that some might include the quotes ("). Should I remove them?

Reply:
LICENSE_KEY: <license key>
NETMAKER_TENANT_ID: <tenant id>

Reply:
but this is an env file :)